samples/precache/README.md

```diff
@@ -3,10 +3,7 @@ This directory contains samples for manual standalone pre-caching testing.
 All the commands provided here must be executed from the main project directory with KUBECONFIG environment variable initialized and pointing to the ACM hub cluster.
 ## Mocks and workarounds ##
 1. Deploy the CRD from [config/crd/bases/ran.openshift.io_clustergroupupgrades.yaml](../../config/crd/bases/ran.openshift.io_clustergroupupgrades.yaml)
-2. If the CSV object is not present on the hub, it must be mocked. 
-3. Copy the autogenerated CSV from [bundle/manifests/cluster-group-upgrades-operator.clusterserviceversion.yaml](../../bundle/manifests/cluster-group-upgrades-operator.clusterserviceversion.yaml). 
-4. Change the namespace to `openshift-operators` and adjust the pre-caching image pull spec to your desired location. 
-5. Deploy the mock policies and objects by
+1. Deploy the mock policies and objects by
```

Comment:
You are numbering 1 twice in line 5 and 6 ?

Reply:
I learnt that when doing a numbered list in markdown, numbers are automatically incremented upon rendering. So you basically add `1.` for every numbered list item, and correct numbers magically appear in the browser

Reply:
ORLY? sounds good, didn't know that :-)